JSZVCR/Classes/Recorder/JSZVCRRecorder.m

```diff
@@ -84,6 +85,7 @@ - (void)recordTask:(NSURLSessionTask *)task didReceiveResponse:(NSURLResponse *)
             NSLog(@"already had response: %@", recording.response);
         }
         recording.response = [JSZVCRResponse responseWithResponse:response];
+        recording.data = nil;
```

Comment:
Again assuming the private API here is similar to similarly-named public ones, I think this will interfere with multipart mime responses (apparently they call `didReceiveResponse:` once for each part). I'm guessing that each part gets at least one  `didReceiveData:` as well, so probably before the change these weren't handled correctly either, but it might be something to verify and document if that's correct.